microsoft-edge/web-platform/web-platform.md

```diff
@@ -18,8 +18,9 @@ There are many tips and tricks to be aware of when building websites or web apps
 *  Detecting Microsoft Edge from your website.
 *  Detecting Windows 11 by using User-Agent Client Hints.
 *  Customizing the **password reveal** button.
+*  Viewing formatted JSON responses and files in browser tabs.
 
-These aspects of developing for the web platform are described below.
+These aspects of developing for, and with, Microsoft Edge are described below.
```

Comment:
@codepo8 - is this positioning ok, "developing for ... Microsoft Edge"?  or too narrow?  It includes "developing with", to soften.

Reply:
this thread can be Resolved, b/c top of article's list now has
*  The JSON viewer matches your operating system's dark or light theme.